configure.ac

```diff
@@ -118,6 +118,10 @@ PKG_CHECK_MODULES([SNDFILE], [sndfile >= 1.0.2])
 AC_SUBST([SNDFILE_CFLAGS])
 AC_SUBST([SNDFILE_LIBS])
 
+PKG_CHECK_MODULES([LIBSYSTEMD], [libsystemd])
+AC_SUBST([LIBSYSTEMD_CFLAGS])
+AC_SUBST([LIBSYSTEMD_LIBS])
```

Comment:
Please not make it mandatory, to fix build on systems without systemd.

Reply:
Also, please add the libsystemd dependency to `.github/workflows/*.yml` so CI can test the code.

Reply:
Does this look valid? Though then I assume the `AM_CONDITIONAL(HAVE_SYSTEMD, ...)` down below become unnecessary?

Reply:
that's valid, but setting an environment variable in `configure.ac` won't make it available in `Makefile.am` and `.c` files. The canonical way is usually something like

```
PKG_CHECK_MODULES([LIBSYSTEMD], [libsystemd], [have_systemd="yes"], [have_systemd="no"])
if test x$have_systemd = xyes; then
    AC_DEFINE([HAVE_SYSTEMD], [1], [Define to 1 if you have the systemd library])
fi
AM_CONDITIONAL(HAVE_SYSTEMD, [test x$have_systemd = xyes])
```